### ISSUE TITLE
Command Line wrapper placement

### ISSUE THREAD

**Issue by mghenderson** [anl-cyberscience/FlexTransform#3]
Alright, this one isn't horribly complicated but I wouldn't mind some debate.  The command-line options are currently handled by **FlexT.py**, which is a wrapper for **FlexibleTransform.py**, which handles the python calls throughout the package.  When the original package was released, **FlexibleTransform.py** also had limited support fort the command-line options, which I'm thinking were removed when Andrew restructured the package last summer.  The problem is where the new **FlexT.py** doesn't get included in the package in the new distribution as it isn't actually in the package (one level too high).  @ChrisStrasburg made the point that there is problem a way to include it using some python/setup.py magic, but given that every relative path would be broken because the same directory structure doesn't exist in the install directory the issue still wouldn't be fixed.  However, there had to be a reason that Andrew moved it up a level, but I can't seem to find anything in the PEP8 docs, or elsewhere online.

@anl-mcampos: Thoughts? 
## Possible Solutions
-  Drop **FlexT.py** one level deeper, keep **FlexibleTransform.py** where it is, and fix the relative references
-  Drop **FlexT.py** & **FlexibleTransform.py** one level deeper each, fix relative references. _(Only considering this to keep the CLI & Python API on different levels, but I'm not leaning towards this one)_
-  Add the command-line handling to **FlexibleTransform.py** and ditch **FlexT.py**
### Old

_(I'm guessing a little about this, as I only have the package after it was built)_

```
FlexibleTransform/
  FlexibleTransform/
    __init__.py
    Configuration/
    OntologyOracle/
    resources/
    SchemaParser/
    Syntax Parser/
    test/
    FlexT.py
    FlexTransform.py
  readme.md
  setup.py
```
#### New

```
FlexibleTransform/
  FlexibleTransform/
    __init__.py
    Configuration/
    OntologyOracle/
    resources/
    SchemaParser/
    Syntax Parser/
    test/
    FlexTransform.py
  FlexT.py
  readme.md
  setup.py
```

**Comment by mghenderson** [comment]
Fixed by PR #4

**Comment by mghenderson** [comment]
I believe I've discovered why Andrew used the original structure.  After moving _FlexT.py_ to within the **FlexTransform** package, this file isn't directly usable (i.e. `python3 FlexTransform/FlexT.py --src ...`).  However, if the FlexTransform pacakge is installed, the `flext` console script command does function correctly (i.e. `flext --src...`).  The error is:

```bash
$ python3 FlexTransform/FlexT.py -h
Traceback (most recent call last):
  File "FlexTransform/FlexT.py", line 7, in <module>
    from FlexTransform import FlexTransform
  File "/Users/mhend/git/FlexTransform/FlexTransform/FlexTransform.py", line 9, in <module>
    from .Configuration import Config
SystemError: Parent module '' not loaded, cannot perform relative import
```
From what I'm finding online, it's occurring because we're using relative imports (or rather because of multiple relative imports).  While I understand that relative pathing is undesirable, it allows FlexTransform to function even when it's not installed as a package.  

Mostly, we need to decide on what functionality we want to support.  Moving to only supporting FlexTransform as a package would simplify a lot of these issues, and allow us to start fixing PEP8 issues.
